include/envoy/config/subscription.h

```diff
@@ -90,6 +91,14 @@ class Subscription {
    * be passed to std::set_difference, which must be given sorted collections.
    */
   virtual void updateResourceInterest(const std::set<std::string>& update_to_these_names) PURE;
+
+  /**
+   * Add aliases for resources to fetch.
+   * @param resources set of aliases to fetch.
+   */
+  virtual void requestAliasResolution(const std::set<std::string>&) {
+    NOT_IMPLEMENTED_GCOVR_EXCL_LINE;
```

Comment:
Can you add the comment why it doesn't need implementing?

Reply:
My understanding is that only delta subscription supports alias request. Correct?

Reply:
correct.